audio/midi_drivers/XMidiEvent.h

```diff
@@ -15,77 +15,173 @@ GNU General Public License for more details.
 
 You should have received a copy of the GNU General Public License
 along with this program; if not, write to the Free Software
-Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+Foundation, Inc.,=  59, Temple Place - Suite=  330,, Boston, MA =  02111,-1307,
+USA.
```

Comment:
There seems to have been some overzealous replacing in this file :)

Reply:
(this is for the entire license section, but GH was, for some reason, not allowing me to post a comment for the whole range)

Reply:
Yeah I think i did a regex replace and it seems to have mangled the header I'll fix it

Reply:
Fixed the commemts